infrastructure/aat.tfvars

```diff
@@ -1 +1,3 @@
 api_test_valid_certificate_thumbprint = "A612B39D42D0D72DEB738F4DF77EF8997023E078"
+
+delete_dispatched_files_cron = "0 0/10 * * * *"
```

Comment:
This should be something more frequent, every ten seconds for example, so that functional tests don't wait too long for files to be deleted.